steps/user-add/step.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -euo pipefail
+
+# NPM credentials (required for commands like `publish`)
+NPM_TOKEN=$(ni get -p '{.npm.token}')
+ORGANIZATION=$(ni get -p '{.npm.organization}')
+USER=$(ni get -p '{.user}')
+ROLE=$(ni get -p '{.role}')
+[ "$ROLE" ] || ROLE="developer"
+
+# Create .npmrc with credentials
+if [ -n "${NPM_TOKEN}" ]; then
```

Comment:
Surely this is required for adding a user to an organization? I don't think you need this check, especially considering the schema structure.

Reply:
Great call. Will update that.